taxi/taxibench_pandas_ibis.py

```diff
@@ -604,6 +553,21 @@ def run_benchmark(parameters):
 
         etl_times_ibis = None
         etl_times = None
+        pd_queries_outputs = {}
```

Comment:
```suggestion
        pd_queries_outputs = {} if parameters['validation'] else None
```
you can do this and then pass this value down below unconditionally